projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.html

```diff
@@ -3,8 +3,12 @@
         <!-- measurement -->
         <tr>
             <td colspan="3" class="measurement">
-                Bandname <i>{{ band.name }}</i
-                ><br />
+                <ng-template [ngIf]="showSingleBandNames">
```

Comment:
you could also just use the @if template https://blog.angular-university.io/angular-if/